dev/com.ibm.ws.security.openidconnect.clients.common/src/com/ibm/ws/security/openidconnect/client/jose4j/util/Jose4jUtil.java

```diff
@@ -205,7 +206,7 @@ private void createWASOidcSession(OidcClientRequest oidcClientRequest, @Sensitiv
         String sid = jwtClaims.getClaimValue("sid", String.class);
         String timestamp = OidcUtil.getTimeStamp();
 
-        String wasOidcSessionId = String.join(":", configId, sub, sid != null ? sid : "", timestamp);
+        String wasOidcSessionId = OidcSessionHelper.createSessionId(configId, sub, sid != null ? sid : "", timestamp);
```

Comment:
Should we just pass the `sid` in as-is and let the `createSessionId()` method decide how to handle a `null` value?

Reply:
i want to keep the null check here, so readers know the sid claim can be null at this level.